oddt/fingerprints.py

```diff
@@ -528,6 +530,30 @@ def combinatorial_rmsd(reference, query):
         return np.sqrt((numla / nula) * (numpa / nupa))
 
 
+def PLECFP(ligand, protein, distance_cutoff=4.5, depth_protein=4, depth_ligand=2, fold_size=16384):
+    """Protein ligand extended connectivity fingerprint"""
+    result = []
+    # removing h
+    protein_dict = protein.atom_dict[protein.atom_dict['atomicnum'] != 1]
+    ligand_dict = ligand.atom_dict[ligand.atom_dict['atomicnum'] != 1]
+    # atoms in contact
+    protein_atoms, ligand_atoms = close_contacts(
+        protein_dict, ligand_dict, cutoff=distance_cutoff)
+    for i, (ligand_atom, protein_atom) in enumerate(zip(ligand_atoms, protein_atoms)):
+        ligand_ecfp = _ECFP_atom_hash(ligand, int(ligand_atom['id']), depth=depth_ligand)
+        protein_ecfp = _ECFP_atom_hash(protein, int(protein_atom['id']), depth=depth_protein)
+        assert len(ligand_ecfp) == depth_ligand + 1
+        assert len(protein_ecfp) == depth_protein + 1
+        if len(ligand_ecfp) < len(protein_ecfp):
+            fillvalue = ligand_ecfp[-1]
+        else:
+            fillvalue = protein_ecfp[-1]
+        for pair in zip_longest(ligand_ecfp, protein_ecfp, fillvalue=fillvalue):
```

Comment:
It is not obvious what is `fillvalue` and why you need it. I think you should add a comment or, even better, explain it in a docstring.